src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-filter/GeneFilter.scss

```diff
@@ -0,0 +1,123 @@
+@import 'src/styles/common';
+@import '../default-transcripts-list/constants';
+
+.container {
+  display: grid;
+  grid-template-columns: $left_column auto;
+  padding: 3px 0;
+}
+
+.filterLabel {
+  grid-column: 1/2;
+  text-align: right;
+  padding-right: 2px;
+}
+
+.expandBtn {
+  margin-left: 10px;
+
+  img {
+    height: 12px;
+    width: 12px;
+  }
+}
+
+.filterBox {
+  display: grid;
+  grid-template-columns: 300px minmax(245px,auto) 20px;
+  background-color: $soft-black;
+  color: $white;
+  width: calc(100% - 320px);
+  height: auto;
+  text-align: left;
+  padding: 15px 15px 45px 20px;
+  margin-top: 2px;
+  margin-left: 2px;
+  font-weight: 300;
+
+  &FullWidth {
+    width: 100%;
+  }
+}
+
+.closeIcon {
+  cursor: pointer;
+  height: 16px; 
+}
+
+.filterContent {
+  display: grid;
+  grid-template-columns: 245px 245px 245px;
+  height: auto;
+}
+
+.header {
+  width: 200px;
+  font-size: 13px;
+  margin-bottom: 10px;
+}
+
+.item {
+  display: block;
+}
+
+.label {
+  color: white;
+  font-size: 12px;
+  line-height: 30px;
+  cursor: pointer;
+  padding-left: 0px;
+  padding-right: 0px;
+  margin-top: 0px;
+}
+
+.radio {
+  height: 12px;
+  width: 12px;
+  border-radius: 50%;
+  border: 1px solid $grey;
+  background-color: none;
+  position: relative;
+  margin: auto;
+  cursor: pointer;
+
+  &Checked {
+    background-color: $white;
+
+    &:after {
+      content: '';
+      position: absolute;
+      top: 2px;
+      left: 2px;
+      width: 6px;
+      height: 6px;
+      border-radius: 50%;
+      background: $green;
+    }
+  }
+}
```

Comment:
Umm... Are all these styles for the radio button necessary? Surely some (`position: relative; cursor: pointer; content: ''; position: absolute;`) are already defined in your RadioButton component?